pom.xml

```diff
@@ -178,6 +184,25 @@
             </compilerArgs>
           </configuration>
         </plugin>
+        <plugin>
```

Comment:
So far we haven't used any test coverage tools. Any particular reasons to add jacoco? If we don't publish any report in GH actions, keep track of the changes in coverage or use any gateway on coverage I see no need to add it. I would like to know your reasoning.